core/trino-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -76,21 +76,29 @@ class StageStatistics extends React.Component<StageStatisticsProps, StageStatist
     }
 
     render() {
+        const query = this.props.query;
         const stage = this.props.stage;
         const stats = this.props.stage.stageStats;
+
+        const blockedTimePercent = 100.0 * parseDuration(stats.totalBlockedTime) / parseDuration(query.queryStats.totalBlockedTime);
```

Comment:
Blocked time is difficult to explain and understand, I would rather not expose this in UI to end users. CPU and scheduled time are good metrics to expose.